modules/data.remote/DESCRIPTION

```diff
@@ -1,13 +1,16 @@
 Package: PEcAn.data.remote
 Type: Package
-Title: PEcAn functions used for remote sensing
+Title: PEcAn functions used for extracting remote sensing data
 Version: 1.7.0
-Date: 2018-12-03
-Authors@R: c(person("Mike","Dietze"))
-Author: Mike Dietze
-Maintainer: Mike Dietze <dietze@bu.edu>
-Description: PEcAn module for processing remote data
+Date: 2019-02-06
+Authors@R: c(person("Mike","Dietze"), person("Bailey", "Morrison"))
+Author: Mike Dietze, Bailey Morrison
+Maintainer: Bailey Morrison <bmorrison@bnl.gov>
+Description: PEcAn module for processing remote data. Python module requirements: requests, json, re, ast, panads, sys. If any of these modules are missing, install using pip install <module name>.
+Depends:
 Imports:
+    MODISTools
+    reticilate
```

Comment:
```suggestion
    reticulate
```

Reply:
@bailsofhay Just need commas after these and you should be good to go.

Reply:
@tonygardella, added the commas!